plugins/inputs/dnac/README.md

```diff
@@ -0,0 +1,78 @@
+
+## Metrics
+
+Client Health API -
+[api doc](https://developer.cisco.com/docs/dna-center/#!get-overall-client-health)
+
+- dnac_client_health
+  - tags:
+    - dnac_host
+    - site_id
+  - fields:
+    - client_type_\<type\>_client_count (type, unit)
+    - client_type_\<type\>_score_value (type, unit)
+    - client_type_\<type\>_score_type\_\<type\>_client_count (type, unit)
+    - client_type_\<type\>_score_type\_\<type>_root_cause\_\<cause\>_client_count (type, unit)
+
+Network Health API -
+[api doc](https://developer.cisco.com/docs/dna-center/#!get-overall-network-health)
+
+- dnac_network_health
+  - tags:
+    - dnac_host
+    - site_id
+  - fields:
+    - overall_health_score (type, uint)
+    - overall_total_count (type, uint)
+    - overall_no_health_count (type, uint)
+    - overall_good_count (type, uint)
+    - overall_fair_count (type, uint)
+    - overall_bad_count (type, uint)
+    - \<device_type\>_health_score (type, uint)
+    - \<device_type\>_total_count (type, uint)
+    - \<device_type\>_bad_count (type, uint)
+    - \<device_type\>_bad_percentage (type, float)
+    - \<device_type\>_fair_count (type, uint)
+    - \<device_type\>_fair_percentage (type, float)
+    - \<device_type\>_good_count (type, uint)
+    - \<device_type\>_good_percentage (type, float)
+    - \<device_type\>_no_health_count (type, uint)
+    - \<device_type\>_no_health_percentage (type, float)
+
+### Example Output
+
+This section shows example output in Line Protocol format.  You can often use
+`telegraf --input-filter dnac --test` or use the `file` output to get
+this information.
+
+```sh
+dnac_client_health, dnac_host=dnac.example.com, host=0d5e9d46cd10, site_id=global client_type_wired_score_type_poor_root_cause_dhcp_client_count=24i, client_type_wired_score_type_poor_client_count=47i, client_type_wireless_score_value=80i, client_type_wireless_score_type_poor_client_count=7i, client_type_wireless_score_type_poor_root_cause_other_client_count=3i, client_type_wireless_score_type_poor_root_cause_association_client_count=3i, client_type_wireless_score_type_fair_client_count=20i, client_type_all_score_value=91i, client_type_wired_client_count=704i, client_type_wired_score_type_poor_root_cause_aaa_client_count=10i, client_type_wired_score_type_fair_client_count=0i, client_type_wired_score_type_idle_client_count=0i, client_type_wired_score_type_nodata_client_count=0i, client_type_wired_score_type_new_client_count=0i, client_type_wireless_client_count=134i, client_type_all_client_count=838i, client_type_wired_score_type_poor_root_cause_other_client_count=13i, client_type_wireless_score_type_poor_root_cause_aaa_client_count=1i, client_type_wireless_score_type_new_client_count=0i, client_type_wireless_score_type_good_client_count=107i, client_type_wireless_score_type_idle_client_count=0i, client_type_wireless_score_type_nodata_client_count=0i, client_type_wired_score_value=93i, client_type_wired_score_type_good_client_count=657i 1635957141000000000

```

Comment:
I think you added spaces between each item for readability? However, in line protocol, there is only a space between the metric name + tags group and the fields and a second space between the field group and the timestamp.

Same with the example below.